website/pages/style.css

```diff
@@ -70,7 +70,7 @@
     margin-top: 72px;
   }
 
-  & > * + *,
+  &:not(.no-section-spacing) > * + *,
```

Comment:
`text-split` components contain their own padding (for now, we plan on removing it soon). So, for now, I've modified this global rule to allow exceptions to the "space children of a `section`", so that the extra spacing could be removed where `text-split` is used in a `section`.